e2e/test/scenarios/models/model-actions.cy.spec.js

```diff
@@ -68,6 +74,17 @@ const SAMPLE_QUERY_ACTION = {
   },
 };
 
+const SAMPLE_WRITABLE_QUERY_ACTION = {
+  ...SAMPLE_QUERY_ACTION,
+  dataset_query: {
+    ...SAMPLE_QUERY_ACTION.dataset_query,
+    native: {
+      ...SAMPLE_QUERY_ACTION.dataset_query.native,
+      query: `UPDATE ${WRITABLE_TEST_TABLE} SET score = 22 WHERE id = {{ ${TEST_TEMPLATE_TAG.name} }}`,
+    }
+  }
+};
```

Comment:
Should we just use this `query` in `SAMPLE_QUERY_ACTION` instead of adding `SAMPLE_WRITABLE_QUERY_ACTION`?

Reply:
I don't feel very strongly one way or the other. I think it's slightly helpful to be testing across a couple different schemas + queries here.